autosklearn/estimators.py

```diff
@@ -95,22 +93,14 @@ def __init__(
             In case of multi-processing, `memory_limit` will be per job.
             This memory limit also applies to the ensemble creation process.
 
-        include_estimators : list, optional (None)
-            If None, all possible estimators are used. Otherwise specifies
-            set of estimators to use.
+        include : dict, optional (None)
+            If None, all possible algorithms are used. Otherwise specifies
+            set of algorithms for each added component is used.
```

Comment:
Is it possible to add that include and exclude are incompatible if used together on the same component? For example, before it use to say:
```
Incompatible with include_estimators.
```

Reply:
Done.